Menu/Tab/index.jsx

```diff
@@ -82,6 +85,16 @@ export default React.createClass({
     }
   },
 
+  componentWillUnmount () {
+    window.removeEventListener('resize', debounce(this.resizeListener))
```

Comment:
This won't work, since the reference to the callback won't be the same. In the constructor, you'll have to create an instance variable for the debounced listener and use that here and in `componentDidMount`.